internal/controller/postgrescluster/pki_test.go

```diff
@@ -44,10 +40,6 @@ import (
 )
 
 func TestReconcileCerts(t *testing.T) {
```

Comment:
Would it be a pain to write a comment for this test stating
* what the test cases are here
* what test case is being covered in the kuttl test <insert path to kuttl test folder here>
?

Reply:
Generally, I don't think we comment the test functions, but it may be worth adding a note in this case since we're pulling out a subset of the original test. I'll see what I can come up with.